model/alumnoDAO.php

```diff
@@ -106,7 +106,10 @@ public function read()
     {
       $id_escuela = $_SESSION['id_escuela'];
       require_once 'alumnoDTO.php';
-      $query = "SELECT alumno.*, usuario.* from alumno alumno, escuela escuela, usuario usuario, director director WHERE usuario.id_usuario = alumno.id_usuario and alumno.id_escuela = escuela.id_escuela and director.id_escuela = escuela.id_escuela and director.id_escuela and alumno.id_escuela and director.id_escuela = '" . $id_escuela . "'";
+      $query = "SELECT alumno.*, usuario.* from alumno alumno, escuela escuela, usuario usuario, director director 
+      WHERE usuario.id_usuario = alumno.id_usuario and alumno.id_escuela = escuela.id_escuela 
+      and director.id_escuela = escuela.id_escuela and director.id_escuela and alumno.id_escuela 
```

Comment:
la consulta está mal, revísala tiene and y and and....
Pero igual a que?
Revísalo por favor

Reply:
Esas correcciones las hice en otra rama de datos duplicados.